src/application/modules/cb_query.py

```diff
@@ -1,9 +1,12 @@
 from cb_utilities import *
+import urllib
```

Comment:
might want to look at refactoring this into urllib2 like the rest of the functions. I can help you with that.

Reply:
Will open an issue. Can close now